.github/workflows/lambda-functions.yml

```diff
@@ -151,32 +151,27 @@ jobs:
           zip -j pinpoint_inbound_sms_lambda pinpoint_inbound_sms/pinpoint_inbound_sms_lambda.py
           aws lambda update-function-code --function-name project-${{ inputs.environment }}-pinpoint-inbound-sms-lambda --zip-file fileb://pinpoint_inbound_sms_lambda.zip
 
+        # This function does not need to package psycopg2-binary, which is included as a lambda layer.  The AWS Lambda execution environment should include boto3 by default.
       - name: Package and deploy VA Profile opt-in/out lambda function
         if: ${{ (inputs.lambdaName == 'ProfileOptInOut') || (inputs.lambdaName == 'All') }}
         run: |
           cd va_profile/
           python3 -m venv venv_va_profile_opt_in_out
           source venv_va_profile_opt_in_out/bin/activate
           pip install -r requirements-lambda.txt
+          pip uninstall -y psycopg2-binary
```

Comment:
We shouldn't install it just to remove it. There's got to be a better way.

Reply:
I could manually run the installation commands for the other dependencies . . .

Reply:
Does it work if psycopg2-binary isn't in the requirements file? I thought Lambda has awareness regarding modules that are in the environment.

Reply:
Yes, that would work, but it would be contrary to the purpose of a requirements file.  There are only 2 dependencies to add.  I will add both directly.

Reply:
I don't know what's best, but install just to uninstall didn't seem right. Thank you for looking into it.